include/wx/msw/private/dpiaware.h

```diff
@@ -0,0 +1,72 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        wx/msw/private/dpiaware.h
+// Purpose:     AutoSystemDpiAware class
+// Author:      Maarten Bent
+// Created:     10/6/2016
```

Comment:
Extremely minor, but I'd prefer to standardize on ISO 8601 format (YYYY-MM-DD) everywhere. In this particular case, I don't even know which date does this refer to, really.